packages/generators/addon-generator.ts

```diff
@@ -39,12 +39,9 @@ export default function addonGenerator(
 		public copy: (value: string, index: number, array: string[]) => void;
 		public copyTpl: (value: string, index: number, array: string[]) => void;
 
-		public prompting(): Promise<{}> {
-			return this.prompt(prompts).then(
-				(props: InquirerScaffoldObject): void => {
-					this.props = props;
-				}
-			);
+		public async prompting(): void {
```

Comment:
As the test said, it should return `Promise<void>`. Strange, the IDE the have told you

Reply:
Strange too. 🤔 

Resolving.